21. Merge Two Sorted Lists.py

```diff
@@ -0,0 +1,37 @@
+# Definition for singly-linked list.
```

Comment:
For this question I prefer the recursive solution because I think its more elegant. But this is ok too. (note you did recursive soln in past)